pandas/compat/pickle_compat.py

```diff
@@ -119,7 +119,7 @@ def load_reduce(self):
         "TimedeltaIndex",
     ),
     ("pandas.indexes.numeric", "Float64Index"): (
-        "pandas.core.indexes.numeric",
+        "pandas.core.indexes.base",
```

Comment:
if these classes dont exist anymore, should we just remove these dict entries?

Reply:
These are a lot of other backward-compat here also, and I think there should be a common decision for all of these that are older than some determined release version. For example, remove everything older than v1.0.

I could ake a seperate issue, where that is discussed?

Reply:
> I could make a separate issue, where that is discussed?

@jreback any idea?  i think this is pickle-related, which i usually punt on